test/e2e/components/vl-map-search.js

```diff
@@ -59,7 +59,7 @@ class VlMapSearch extends VlElement {
 
```

Comment:
Er is geen specifieke e2e test omdat alles door bestaande map search e2e test wordt gedekt, klopt?
Idem demo pagina neem ik aan.

Reply:
De component is nu binnen de vl-map apart getrokken omdat deze toevallig gebruikt werd in een specifieke context van een reeds bestaande applicatie. De vl-map-search was initieel niet ontworpen om te gebruiken in een formulier. Deze inspanning is puur om in die context een upgrade te kunnen uitvoeren. Zodra uit nieuwe applicaties blijkt dat deze component vaak voorkomt, zal deze afgesplitst worden en uiteraard ook voorzien worden van afzonderlijke e2e testen. In kader van de vl-map behouden we de testen op vl-map-search niveau.